src/SourceBuild/content/repo-projects/fsharp.proj

```diff
@@ -19,9 +19,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <RepositoryReference Include="arcade" />
-    <RepositoryReference Include="msbuild" /> 
-    <RepositoryReference Include="source-build-externals" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
+    <RepositoryReference Include="arcade" />  
```

Comment:
nit:

```suggestion
    <RepositoryReference Include="arcade" />
```

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/19bc3f3db2d22938fd444d2423fa4e3f57029d7c